vcl_templates/assets.vcl.erb

```diff
@@ -153,6 +153,11 @@ sub vcl_recv {
      error 801 "Force SSL";
   }
 
+  # Serve and log an empty response for analytics purposes
+  if (req.url ~ "^/static/a") {
```

Comment:
Also a good point, and fixed. In reality it will only be called with a querystring, so it now must begin with `/static/a?`. 

If someone calls it with `/static/a`, it will fall through to the backend and 404.